fix.sh

```diff
@@ -12,6 +12,12 @@ xmlstarlet ed -P -L \
 	-u '//*[@glib:error-domain="g-option-context-error-quark"]/@glib:error-domain' -v g-option-error-quark \
 	GLib-2.0.gir
 
+# incorrectly assumes GdkEventButton when other variants may be passed
+xmlstarlet ed -P -L \
+	-u '//glib:signal[@name="icon-press"]//_:parameter[@name="event"]/_:type[@name="Gdk.EventButton"]/@name' -v "Gdk.Event" \
+	-u '//glib:signal[@name="icon-release"]//_:parameter[@name="event"]/_:type[@name="Gdk.EventButton"]/@name' -v "Gdk.Event" \
```

Comment:
Might be useful to add another condition at the beginning so this only ever goes into `GtkEntry` and not anything else that might have the same signal name in the future.

Reply:
Done.